aws_lambda_powertools/utilities/_data_masking/constants.py

```diff
@@ -1,5 +1,9 @@
 DATA_MASKING_STRING: str = "*****"
+# The maximum number of entries that can be retained in the local cryptographic materials cache
 CACHE_CAPACITY: int = 100
+# The maximum time (in seconds) that a cache entry may be kept in the cache
 MAX_CACHE_AGE_SECONDS: float = 300.0
-MAX_MESSAGES_ENCRYPTED: int = 200
-# NOTE: You can also set max messages/bytes per data key
+# Maximum number of messages which are allowed to be encrypted under a single cached data key
+MAX_MESSAGES_ENCRYPTED: int = 4294967296  # 2 ** 32
+# Maximum number of bytes which are allowed to be encrypted under a single cached data key
+MAX_BYTES_ENCRYPTED: int = 9223372036854775807  # 2 ** 63 - 1
```

Comment:
We need to review these values and find the right balance between the user experience and best practices.

Reply:
These are the default values from the [SDK](https://github.com/aws/aws-encryption-sdk-python/blob/93a67d8a3806f560ead950e6d8898e53c4c4f9df/src/aws_encryption_sdk/internal/defaults.py#L39-L41), would it be inconvenient for the customer to use these values as opposed to the previous ones? If so, we can go back to the previous ones, I just got them from an example from the docs. But [best practices](https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/best-practices.html) do say that we should use default values.

Reply:
@heitorlessa @sthulb Do we want to change these values for `max_messages_encrypted` and `max_bytes_encrypted`? The current values are the default ones that came from the encryption SDK docs page, but it could be a security concern bc cxs would be using one cached key to encrypt 4294967296 messages by default... I was thinking we use lower numbers (maybe `200` and `2000` respectively) and explain the range of possible values they take.

Reply:
Left the eSDK defaults and explained in the docs how to override them